src/fmu/tools/rms/copy_rms_param_to_ertbox_grid.py

```diff
@@ -134,6 +137,9 @@ def copy_rms_param(params: dict) -> None:
     required_kw_list = ["Mode"]
     check_missing_keywords_list(params, required_kw_list)
     mode = params["Mode"]
+    debug_level = DEBUG_ON
```

Comment:
In general, is this  DEBUG meant for developer of end.-user? If developer, using the built-in python logging system (with info, debug modes) is much more neat. See e.g. 

https://github.com/equinor/fmu-tools/blob/main/src/fmu/tools/domainconversion/dconvert.py

Reply:
Meant for adjusting log output to the user. A revision of this should be done when this is done in the APS code also. Currently the different log levels (debug_levels) is defined by an int in this function and by an enum (int) in APS.